multicluster/charts/linkerd-multicluster-link/templates/psp.yaml

```diff
@@ -1,8 +1,9 @@
+{{if .Values.enablePSP -}}
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: multicluster-link-psp
+  name: multicluster-link-psp-{{.Values.targetClusterName}}
```

Comment:
nit: should we prefix the name with `linkerd` because we seem to do that everywhere else? 🤔

Reply:
Yeah, think the base mc PSP rolebinding also missed the `linkerd` prefix so I changed both rolebinding resources to make it consistent.